src/main/java/core/mate/academy/model/Truck.java

```diff
@@ -1,13 +1,14 @@
 package core.mate.academy.model;
 
-/**
- * Add some custom fields that could be only in Truck
- * Do not remove no-field constructor
- */
 public class Truck extends Machine {
```

Comment:
Consider adding access modifiers for your custom fields in the `Truck` class. The fields from the superclass `Machine` should be private and accessed through getters and setters for better encapsulation.